aten/src/ATen/Utils.cpp

```diff
@@ -56,7 +57,7 @@ Tensor empty_generic(
   // technically this can be inferred from the device, but usually the
   // correct setting is obvious from the call site so just make callers
   // pass it in
-  c10::DispatchKey dispatch_key,
+  c10::DispatchKeySet ks,
```

Comment:
> I included more info on benchmarking in the next PR, where I added a bunch of optimizations.

I don't see benchmarks but maybe I'm not looking hard enough

Reply:
Whoops, I must have fudged ghstack a while ago (there are two copies of the PR floating around and the other one has the description, but for some reason the diff mapped to this PR has the correct description in fbcode).

I moved the description to this PR and closed the duplicate (https://github.com/pytorch/pytorch/pull/70356)